src/Files.App/Strings/en-US/Resources.resw

```diff
@@ -2601,6 +2601,12 @@
   <data name="ApplyToAllConflictingItems" xml:space="preserve">
     <value>Apply this action to all conflicting items</value>
   </data>
+  <data name="OpenTerminal" xml:space="preserve">
+    <value>Open in terminal</value>
+  </data>
+  <data name="OpenTerminalAsAdmin" xml:space="preserve">
+    <value>Open in terminal as admin</value>
```

Comment:
```suggestion
    <value>Open in terminal as administrator</value>
```

Reply:
I won't be able to fix that until Saturday